app/scripts/modules/core/instance/instanceListBody.directive.js

```diff
@@ -103,17 +103,19 @@ module.exports = angular.module('spinnaker.core.instance.instanceListBody.direct
           switch(filterType) {
             case 'id':
               return a.id.localeCompare(b.id);
-            case 'launchTime':
+            case 'launchTime': {
```

Comment:
I've never seen curly braces used on `case` blocks - is the linter suddenly complaining about this?